### ISSUE TITLE
wrong accuracy of resnet18

### ISSUE THREAD

**Issue by marsggbo** [huyvnphan/PyTorch_CIFAR10#18]
I test resnet18 on the CIFAR10 test set, the accuracy is 81.24%, but the reported acc is over 93.07%

Is my test code wrong?

The code is below:

```python
import torch
from torch.utils.data import DataLoader
from torchvision import datasets, transforms

from model import *


if __name__ == '__main__':
    net = resnet18(pretrained=True, device='cuda').cuda()
    net.load_state_dict(torch.load('state_dicts/resnet18.pt'))

    mean = (0.4914, 0.4822, 0.4465)
    std = (0.2471, 0.2435, 0.2616)
    t = transforms.Compose([
        transforms.ToTensor(),
        transforms.Normalize(mean, std),
    ])
    dataset = datasets.CIFAR10(root='.', train=False, transform=t, download=True)
    dataloader = DataLoader(dataset, batch_size=10, num_workers=0, shuffle=False)
    acc = 0.
    preds = []
    labels = []
    for i, (img, label) in enumerate(dataloader):
        # if i < 10:
        if 1:
            img = img.cuda()
            pred = net(img).detach().cpu()
            pred_indices = pred.argmax(1)
            preds.append(pred_indices)
            labels.append(label.cpu())
    preds = torch.cat(preds)
    labels = torch.cat(labels)
    print(sum(preds==labels)/len(preds))

```

**Comment by huyvnphan** [comment]
You have to put your model in eval mode first: 
`net = resnet18(pretrained=True, device='cuda').eval().cuda()`
